icons/windows95.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Windows 95 icon</title><path d="M5.7122 1.5956l-.756.068-.238.55.734-.017zm1.3901.927l-.978.137-.326.807.96-.12.345-.824zm-2.2131 1.012l-.72.05-.24.567.721-.017zm3.7242.309l-1.287.068-.3941.9611 1.27-.053zm1.87.566l-1.579.069-.566 1.3571 1.596-.088.548-1.338zm-4.1881.037l-.977.153-.343.8061.976-.12zm6.1432.6681l-1.87.135-.636 1.527 1.87-.154zm2.9251.219c-.11 0-.222 0-.334.002l-.767 1.8511c1.394-.03 2.5201.088 3.3731.38l-1.748 4.2002c-.955-.304-2.0821-.444-3.3602-.394l-.539 1.305a8.7624 8.7624 0 013.3641.396l-1.663 4.0142c-1.257-.27-2.3811-.395-3.3862-.344l-.782 1.887c3.3621-.446 6.3473.8221 9.0084 3.7732L24 9.23c-2.3251-2.5751-5.2002-3.8792-8.6374-3.8962zm-.644.002l-2.024.12-.6871 1.6801 2.025-.189zm-10.6024.05l-.719.036-.224.566h.703l.24-.601zm3.6901.397l-1.287.069-.395.959 1.27-.05zm-2.268.517l-.9941.154-.344.8071.979-.121zm4.1371.066l-1.58.069-.567 1.3351 1.596-.085.55-1.32zm1.955.688l-1.871.1351-.635 1.527 1.887-.154zm2.2812.1901l-2.0101.136-.7 1.682 2.04-.19.67-1.629zm-10.5685.066l-.739.035-.238.564h.719l.258-.6zm3.7052.293l-1.303.085-.394.961 1.287-.035zm11.8385.255a6.7183 6.7183 0 012.7771 1.7171l-1.75 4.2372c-.6171-.584-1.1501-.961-1.6111-1.149l-1.201-.498zm-14.1546.362l-.976.154-.344.807.961-.12.36-.841zm4.1862 0l-1.594.052-.5491 1.3541 1.594-.086zm1.957.668l-1.887.152-.619 1.5081 1.87-.135.636-1.527zm2.2461.275l-2.006.12-.703 1.6652 2.041-.156zm-10.6024.104l-.718.033-.24.5491.718-.016zm3.7252.2731l-1.289.07-.4101.961 1.287-.03.412-1zm1.87.6l-1.596.05-.55 1.356 1.598-.084.547-1.322zm-4.1861.037l-.979.136-.324.805.959-.119zm6.1402.633l-1.87.154-.653 1.5271 1.906-.154zm2.2661.275l-2.025.12-.6861 1.6631 2.025-.172zm-10.5684.031l-.739.037-.238.565.72-.016zm3.6731.362l-1.289.068-.41.9781 1.305-.05zm-2.285.5331l-.9761.154-.326.805.96-.12.342-.84zm4.1531.07l-1.596.066-.565 1.356 1.612-.084zm1.957.666l-1.889.154-.617 1.5261 1.886-.15zm2.2802.223l-2.0251.12-.685 1.6651 2.041-.172.67-1.613zM.961 13.02l-.738.053L0 13.64l.72-.02.24-.6zm3.7052.31l-1.285.07-.395.976 1.287-.05.393-.997zm11.9225.07c1.08.29 2.024.821 2.8141 1.6131l-1.715 4.1832c-.8921-.754-1.8201-1.32-2.8142-1.664l1.715-4.1332zm-10.0354.515L4.9563 14l-.5491 1.3201 1.578-.066.567-1.338zm-4.1842.014l-.996.156-.309.7891.961-.105zm6.1403.6701l-1.9041.154-.617 1.527 1.889-.154.633-1.527zm2.231.324l-2.025.123-.686 1.682 2.026-.174zm-6.8632.328l-1.3.068-.3971.979 1.285-.053zm1.871.584l-1.578.068-.566 1.3341 1.595-.064zm1.9531.701l-1.867.137-.635 1.5101 1.87-.137zm2.2301.31l-2.005.1221-.7031 1.68 2.04-.19.6691-1.611z"/></svg>
```

Comment:
Are you able to reduce this down to 3 decimals?
```svg
M12.008 7.137l.686-1.681 2.024-.12-.686 1.611-2.024.19zm1.905.104l-2.008.137-.701 1.682 2.04-.189.669-1.63zm-.789 1.922l-2.006.118-.705 1.666 2.041-.156.67-1.628zm-.788 1.92l-2.025.121-.685 1.664 2.024-.172.686-1.613zm-.789 1.886l-2.025.121-.687 1.665 2.041-.172.671-1.614zm-.807 1.956l-2.025.122-.685 1.681 2.024-.172.686-1.631zm-2.815 2.044l-.703 1.679 2.04-.19.67-1.611-2.007.122zm3.877-10.346l.636-1.508-1.87.135-.636 1.528 1.87-.155zm-.17.429l-1.87.136-.635 1.527 1.887-.153.618-1.51zm-.756 1.835L8.99 9.04l-.618 1.509 1.869-.136.635-1.526zm-.807 1.922l-1.869.155-.653 1.526 1.905-.155.617-1.526zm-.804 1.938l-1.889.153-.617 1.526 1.886-.15.62-1.529zm-.755 1.852l-1.905.156-.617 1.526 1.888-.155.634-1.527zm-2.676 2.074l-.636 1.51 1.87-.137.634-1.509-1.868.136zM9.933 5.748l.549-1.337-1.578.068-.565 1.356 1.594-.087zM7.532 7.772l1.595-.086.551-1.322-1.58.069-.566 1.339zm1.389.446l-1.595.053-.549 1.354 1.595-.085.549-1.322zm-2.95 3.328l1.597-.084.547-1.322-1.595.05-.549 1.356zm1.337.534l-1.595.067-.566 1.355 1.612-.084.549-1.338zm-.756 1.835L4.957 14l-.549 1.32 1.578-.067.566-1.338zm-.804 1.921l-1.578.068-.566 1.335 1.595-.065.549-1.338zM8.201 4.82l.412-.977-1.287.069-.394.961 1.269-.053zM6.126 6.811l1.269-.049.412-.979-1.287.068-.394.96zm-.773 1.838l1.288-.035.41-1.012-1.303.086-.395.961zm-.807 1.921l1.288-.032.411-.998-1.288.069-.411.961zm-.804 1.954l1.304-.052.395-.996-1.288.069-.411.979zm.925.807l-1.286.069-.396.978 1.288-.052.394-.995zm-2.091 1.988l-.397.979 1.287-.052.412-.994-1.302.067zM7.103 2.523l-.977.138-.327.806.961-.12.343-.824zM4.975 5.407l.977-.121.343-.84-.977.153-.343.808zm-.772 1.852l.978-.12.361-.839-.996.153-.343.806zM3.414 9.18l.96-.121.361-.842-.978.156-.343.807zm-.787 1.937l.959-.118.344-.823-.979.137-.324.804zm-.773 1.852l.959-.12.343-.841-.978.155-.324.806zm-.481 1.116l-.309.79.961-.105.343-.84-.995.155zm4.34-12.489l-.756.069-.239.549.736-.018.259-.6zm-.824 1.94l-.72.05-.24.567.721-.018.239-.599zm-.772 1.851l-.719.034-.224.567h.703l.24-.601zm-.772 1.921l-.738.036-.239.565h.72l.257-.601zm-.824 1.958l-.719.033-.24.549.72-.016.239-.566zm-.754 1.85l-.737.036-.24.566.721-.016.256-.586zM0 13.641l.721-.02.24-.601-.738.053-.223.568zm24-4.41l-5.471 13.173c-2.661-2.952-5.645-4.219-9.006-3.773l.781-1.886c1.005-.051 2.128.071 3.386.342l1.663-4.014a8.751 8.751 0 00-3.364-.395l.54-1.304c1.277-.05 2.403.09 3.358.394l1.749-4.202c-.852-.291-1.979-.409-3.373-.378l.767-1.851c3.584-.068 6.57 1.235 8.97 3.894zm-4.598 5.782a6.161 6.161 0 00-2.814-1.613l-1.715 4.133c.994.344 1.922.91 2.815 1.664l1.714-4.184zm2.265-5.439a6.711 6.711 0 00-2.779-1.716l-1.784 4.306 1.2.498c.462.187.994.567 1.611 1.15l1.752-4.238z
```

Reply:
Done 💻